### PR TITLE
Replaced images for RHAMT rebranding

### PR DESCRIPTION
**Pull request by RichardHoch**
Document preview: http://file.emea.redhat.com/~rhoch/06252020/CLI/

Replaced 3 obsolete images with those found in https://issues.redhat.com/browse/WINDUP-2559.

### CODE REVIEW

**Comment by RichardHoch** [comment]
@apinnick Please review.

**Comment by apinnick** [comment]
I see a screenshot that needs to be updated. Since I don't see the master file here, I don't know the name of the file. The number is "Figure 4.1. Issues Report with CSV Download" in "4.1. Export the Report"

File: docs/topics_5/review-reports.adoc [hunk 1]
```diff
@@ -285,23 +285,23 @@ image::shared-archives-app-list.png[Archives Shared by Multiple Applications]
...
-== Rule Provider Execution Overview
+== Rule Providers Execution Overview
 
 Access this report from the report landing page by clicking the *Rule providers execution overview* link.
 
 This report provides the list of rules that executed when running the {ProductShortName} migration command against the application.
 
-.Rule Provider Execution Overview
+.Rule Providers Execution Overview
 image::report-jee-example-ruleprovider.png[Rule Provider Execution Overview]
```
Comment (apinnick) [inline docs/topics_5/review-reports.adoc:295]: Update image name as well. Not a big thing, but should be consistent.

**Comment by apinnick** [comment]
@RichardHoch  Please squash commits

**Pull request merged**